samcli/lib/build/workflow_config.py

```diff
@@ -74,6 +72,29 @@ class UnsupportedRuntimeException(Exception):
     pass
 
 
+def get_layer_subfolder(runtime):
+    subfolders_by_runtime = {
```

Comment:
how did we arrive at this?

Reply:
you mean subfolders? I simply copied from here: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-path